plugin/storage/cassandra/spanstore/writer.go

```diff
@@ -65,6 +65,13 @@ const (
 	durationBucketSize = time.Hour
 )
 
+const (
+	indexAndStore = storageMode(iota)
+	indexOnly     = storageMode(iota)
+	storeOnly     = storageMode(iota)
```

Comment:
this is not the iota pattern, see https://dlintw.github.io/gobyexample/public/constants-and-iota.html

Reply:
e.g.
```
const (
	Spring = Season(iota)
	Summer
	Autumn
	Winner
)
```